refactor: improve BSS and FAQ doc (#152)
mia-platform/documentation: docs/fast_data/bucket_storage_support/overview.md

## Additional Components

Bucket Storage Support is made so that other services can use the files written into the bucket according to their own business logic.
Components that work on the bucket stored files are identified as Data Organizer plugins.
In the following sections are reported the Data Organizer services that are already offered out of the box and that can be attached
directly to the events emitted by the Ingestion Storer.

## Architecture Example

> In the below image you can see diagram of a possible architecture that employs all three Bucket Storage Support components.

Reviewer: ```suggestion
The image below shows a diagram of a possible architecture that employs all three Bucket Storage Support components.
```
